src/commtrack.js

```diff
@@ -314,14 +329,21 @@ define([
                 src: { presence: "notallowed" },
                 dest: {
                     validationFunc: function (mug) {
-                        if (mug.p.dest.value) {
+                        if (mug.p.dest) {
                             return 'pass';
                         }
                         return 'Receive must have a Destination Case.';
                     },
                 },
             }
-        });
+        }),
+        setValuePaths = _.chain(setvalueData)
+            .map(function (mugSetValues, mugClass) {
+                return _.map(mugSetValues, function(attrs) {
+                    return attrs.path;
```

Comment:
Not sure it matters, but might want to do `RegExp.escape(attrs.path)` to be safe.